### DISCUSSION TITLE
Writing Async Cross-Chain Smart Contracts with Agoric - Nebular 2024 Workshop - Brussels

### DISCUSSION THREAD

**Discussion by Jovonni** [Agoric/agoric-sdk#9689] [Show and tell]
[workshop slides](https://docs.google.com/presentation/d/1Zh8t5dDQTknc3wMfDm0MGHRyGLgzI_7r4BlG4GicPRM/edit#slide=id.g2e9ef5c81f5_0_681)

**Comment by dckc** [comment]
@Jovonni do you have a pointer to recordings?
